game/src/sandbox/dashboards/trip_table.rs

```diff
@@ -585,20 +587,21 @@ fn trip_category_selector(ctx: &mut EventCtx, app: &App, tab: DashTab) -> Widget
     let total = finished + cancelled + unfinished;
 
     let btn = |dash, action, label| {
+        let mut button = ctx.style().btn_primary_light_text(label);
         if dash == tab {
-            Text::from(Line(label).underlined())
-                .draw(ctx)
-                .centered_vert()
-        } else {
-            Btn::plaintext(label).build(ctx, action, None)
+            button = button
+                .disabled(true)
+                .bg_color(ctx.style().btn_primary_light.bg, ControlState::Disabled)
+                .label_styled_text(Text::from(Line(label).underlined()), ControlState::Default)
```

Comment:
We have several set of "tabs" in the app, but are not uniform in how we handle them. Having a go-to "Tab" implementation would be great. 

I haven't looked to see how much of a pain it would be to wire up something that works for all the cases we have. It might be awful.

Reply:
The tabs in the trip table look a bit off to me now, but they're not stranger than the underlined style previously. I've seen a few different tab styles in Figma. I think some of the designs needed to distribute the extra horizontal space evenly across all buttons, and I haven't yet figured out the flexbox tricks for that.

It'd also be nice to have a higher-level API for describing a Panel with multiple tabs directly -- maybe this is what you're referring to here? But right now, the tabs in the info panel are vastly more complicated than the trip table ones, and I can't offhand think of anything other than these two cases.

Reply:
One other place we use "tabs" is the "jump to time" dialog.

There are a handful of other places, where we aren't *exactly* using a "tab" metaphor, but it's similar in that there are several buttons, each of which when clicked, becomes disabled and shows it's corresponding content.

grepping for `.disabled(` will give you an idea. Not *all* of the matches, but several of the form `.disabled(current == idx)` exhibit this behavior.

> It'd also be nice to have a higher-level API for describing a Panel with multiple tabs directly -- maybe this is what you're referring to here?

Yep! That's what I was hoping to one day have.